### ISSUE TITLE
Some thoughts

### ISSUE THREAD

**Issue by vr00n** [Streets-Data-Collaborative/Autonomous_Transportation_Analyzer#3]
BRT is all about improving fixed route transit which amongst transit planners is very very very different from non-fixed route transit (door to door shared vehicle) or microtransit which is what silicon valley tends to love.

via http://humantransit.org/2015/08/uber-discovers-the-inherent-efficiency-of-fixed-routes.html

His basic point is that fixed route transit is, in the net, more efficient than door-to-door and I agree with him.

Therefore, is a key assumption for this project that we are identifying fixed route transit that leverages autonomous driving which makes fixed route less expensive to operate? I'm not fully clear what the policy objectives are.

For big cities - AV fixed route transit means replacing existing bus routes with AV bus routes? I think that's a seriously tough sell in the short-term.

For smaller cities - this means bringing and operating AV fixed route transit to areas which didn't make sense to offer public transit before (aka transit deserts). I think the opportunity lies more here. 

So less LA city proper and more:
- Palmdale
- Santa Fe Springs
- Hawthrone
- El Monte
- Compton
via http://www.onlyinyourstate.com/southern-california/poorest-cities-so-cal/

Location selection can also be to identify transit deserts i.e places that are far away from existing transit but tend to be home/work hubs.

LEHD data (http://onthemap.ces.census.gov/) or the  can be used to identify commuting patterns from these communities.

Combined with https://github.com/argo-marketplace/RadRoads, the top-n most successful pilot sites for autonomous + fixed-route transit can be presented.

That seems to be where my head is at for this.

**Comment by patwater** [comment]
Some thoughts. 

1. Look into optimal fixed routes. There's a lot to go into here. Could use RadRoads perhaps to help inform this? Also perhaps do low / medium / high coverage options with different levels of routing. We could use the Census community flows and also other origin / destination data as well. 

2. @vr00n what are your thoughts on planning for semi-fixed routes? Thinking of fixed routes where the number of automated buses fluctuates up or down based on demand. Seems like that could be an option for dealing with last mile issues. Also opportunity for the whole Silicon Valley network thing to shift route frequency based on demand perhaps?

3. Could also tie into ideally into local SQUID mapping and a maintenance program to deal with lane markings etc. There's some momentum there and looking to apply for a community CfA fellowship to get resources for that.

**Comment by dmarulli** [comment]
Some thoughts.

**Less important:**
Think we need to be careful here. To be honest, that piece strikes me as pretty cloudy with ideology.

Uber's Express POOL isn't quite an example of "cool kids at Uber [...] showing that for all their pretense of having invented something new..." they are evolving "the same solutions that worked in the past."

I think the more objective read is that the solutions didn't work in the past and that's why there was enough demand for Uber to become a company valued at  ~70 billion in spite of in principle inefficiencies of door-to-door transportation. Not to mention the door-to-door option is still available for those who still prefer that option.

Also don't know how familiar you guys are with Express POOL, but to call the routes fixed is a hard stretch imo. You get a suggested pick up point, but that pickup point does not seem to be the same each time--perhaps it depends on which direction the car is coming from and/or is dynamically updated as new data come in. And of course the route itself between origins and destinations isn't fixed, and the car does not have to stop when there is nobody there.

Will resist a "[He's] an idiot" joke though :p. 

**More important:**
All this is to say, I'm not sure us running (or supporting an institution that is willing to run) a static analysis to determine fixed routes for buses is the most valuable use of our time (thought 1.).

Very open to pushback on all this though.

**Comment by vr00n** [comment]
The $70 Billion valuation of Uber is equally ideological. 

regardless of physically fixed or virtually fixed routes...

The bottomline imho is being able to offer an analysis of street networks (OSMnx) and combine it with home/work population centers (LEHD) towards suggesting how a hypothetical autonomous bus can service large groups of people in a given region.

This type of analysis can potentially scale across the nation.

**Comment by patwater** [comment]
Yeah agree on the value of that analysis @vr00n. What do you see as the key steps in executing that V?

**Comment by dmarulli** [comment]
Re: valuation - yeah that's a fair enough critique V

**Comment by vr00n** [comment]
This offers a start where they have mapped at the state level. Would be v interesting to reproduce at a county level.
https://lehd.ces.census.gov/doc/workshop/2017/Presentations/TheaEvans.pdf

![image](https://user-images.githubusercontent.com/4397663/38686222-1999139a-3e31-11e8-8cad-2647b4470e34.png)

**Comment by patwater** [comment]
County level would likely be too agreggate. Would need to be more granular say census block

**Comment by vr00n** [comment]
Integrating this with Geo-crosswalk would be epic because then you could do neighborhood level analysis or for that matter {any-local-geo} level analysis.

**Comment by patwater** [comment]
Strong town comes out in favor of autonomous electric vehicles:
https://www.strongtowns.org/journal/2018/4/9/the-potential-of-autonomous-electric-busses 

_"The primary limiting cost factor for any transit service is driver compensation, not capital or maintenance costs associated with the bus itself. Because autonomous vehicles do not require a driver, transit systems that switch from piloted vehicles to autonomous vehicles can actually support many more buses, vans, trolleys, etc. with the same budgets they currently employ."_

...

_"A research paper authored by Neil Quarles and Kara M. Kockelman of the University of Texas at Austin sheds light on the cost savings AEVs could provide Austin’s Capital Metro Transportation Authority (known locally as and henceforth shortened to: “CapMetro”). This article will use their findings as a springboard for discussion on the future of public bus transportation in an AEV future.

(Note: In their research, Quarles and Kockelman estimate that CapMetro will replace about 36 buses per year out of their fleet of 438, and their long-term findings are based on this. For the simplicity of this article, I presume that CMTA has chosen to replace all of its standard buses with AEV buses in the year 2028.)

According to Quarles and Kockelman, the contracted companies who manage CapMetro’s bus drivers consume about 45% percent of CapMetro’s operating budget. Driver compensation, management, and overhead costs CapMetro an average of $272K per bus per year, and $3.26M per bus over a 12 year “lifetime” for each bus (K = thousand and M = million). Automated buses could eliminate these driver costs completely and save CapMetro a maximum of $3.26M per bus lifetime. However, we must also consider the costs of adding automated technology to existing bus platforms. Using conservative estimates of $80K in capital costs for adding automated technology to a bus from the factory, CapMetro’s potential yearly savings reduce to $265K per bus per year, and to $3.18M per bus lifetime.

438 buses x $265K (savings per bus) = $116 million in savings each year

CapMetro has 438 buses, so $265K x 438 equates to $116M in savings for CapMetro per year for its bus fleet, or about $1.39B (billion) in savings for the 12-year lifetime of the fleet. Keep in mind, these are just the savings from making the buses autonomous. Thus far, we’re assuming they still run on diesel fuel. Now let’s see what happens when we bring electrification into the equation."_

...

**Yields big results:** **_"In 2028, that $1.46B could afford CapMetro many options to improve service. In fact, it would be revolutionary. If CapMetro took all of the $1.46B they save from automation and electrification of their buses, they could conceivably run a fleet of 2.4 thousand full size buses in their service area. "_**

The paper itself is a bit opaque on where it's getting it's cost estimates though aligns with my discussions and research. http://www.ce.utexas.edu/prof/kockelman/public_html/TRB18AeBus.pdf
